website/blog/releases/0.8.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "Apache Fluss 0.8: Streaming Lakehouse with Iceberg/Lance"
+sidebar_label: "Announcing Apache Fluss 0.8"
+authors: [giannis, jark]
+date: 2025-10-30
+tags: [releases]
+---
+
+![Banner](../assets/0.8/banner.jpg)
+
+🌊 We are excited to announce the official release of **Fluss 0.8**!
+
+This is the first ASF release for Apache Fluss (incubating), marking a significant milestone in our journey to provide a robust streaming storage platform for real-time analytics.
+Over the past four months, we’ve delivered lots of improvements and new capabilities, with more than 390+ commits, across the Streaming Lakehouse ecosystem,
+including: deeper integration with Apache Flink, extensive improvements in the Streaming Lakehouse with support for [Apache Iceberg](https://iceberg.apache.org/) and [Lance](https://github.com/lancedb/lance),
+and the introduction of [Delta Joins](https://cwiki.apache.org/confluence/display/FLINK/FLIP-486%3A+Introduce+A+New+DeltaJoin), which redefine efficiency in stream processing.
+
+Apache Fluss 0.8 marks a new era of **real-time**, **unified**, and **zero-state streaming**, designed to power the next generation of data platforms, focusing on performance, scalability, and simplicity of the overall architecture.
```

Comment:
According blog content, how about using specific key words to help user understand easier ?
  
`**zero-state streaming join**, **real-time multimodal AI analytics**, and **stream-batch unification**`